instrument/MERLIN_Parameters_2018_03.xml

```diff
@@ -0,0 +1,478 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<parameter-file instrument = "MERLIN" valid-from = "2017-09-12T11:00:01">
```

Comment:
Is the `valid-from` date correct here? It does not match the year indicated by the filename.

Reply:
Fair point. I have not modified it as, AFAIK, parameters file date is not used anywhere, but it may be used in a future. I have modified it making equal to Defenition date.